src/Shell/ConsoleShell.php

```diff
@@ -7,13 +7,12 @@
  * For full copyright and license information, please see the LICENSE.txt
  * Redistributions of files must retain the above copyright notice.
  *
- * @codeCoverageIgnore
- *
  * @copyright Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
```

Comment:
I think I might be leaning towards maintaining this one-line change to ignore this "boilerplate" class from testing entirely, rather than all of the extra changes below that are required to make it testable (and even then only "mostly" so.)

The other benefit is that importing upstream changes from the cake core's skeleton is much easier since all we have to preserve is a single `@codeCoverageIgnore`. 

It also reduces the number of "copied" files that must be maintained in all projects spawned from our skeleton since there is no counterpart test case to maintain. Thoughts?